doc/source/whatsnew/v0.16.0.txt

```diff
@@ -298,6 +298,82 @@ Backwards incompatible API changes
      p // 0
 
 
+Categorical Changes
+~~~~~~~~~~~~~~~~~~~
+
+.. _whatsnew_0160.api_breaking.categorical:
+
+In prior versions, Categoricals that with had an unspecified ordering (meaning no ``ordered`` keyword was passed) were defaulted to have a lexographic ordering of the values and designated as ``ordered`` Categoricals. Going forward, the ``ordered`` keyword in the ``Categorical`` constructor will default to ``False``, so ordering must now be explicit.
```

Comment:
This should either remove the mention of the "lexi sorted categories" or state that `categories are still lexi-sorted, but the resulting categorical data is considered "unordered" (cat.ordered == False)`